home4.2/ProfProg-home-4.2-task.md

```diff
@@ -0,0 +1,4 @@
+**TASK. Try adapter pattern one more time**
+1. Port code base for adapters to make it work.
+
+2. Propose one example where Adapter pattern can help with program design.
```

Comment:
I don't see an example related to the second part of home task. You can use some already existed example from standard libraries or propose your own scenario when Adapter can simplify code